### ISSUE TITLE
Allow default variables to be passed

### ISSUE THREAD

**Issue by bebraw** [antwarjs/default-theme#4]
I tried overwriting some of the default values through the custom styles hack (primarily content width). I realized it would be a much better idea to be able to provide custom configuration to themes (ie. pass name of file containing variables or json/js conf even). Do you want me to give it a go?

Btw that custom style thing should probably go to a higher level. It's not nice that themes have to inject it.

**Comment by eldh** [comment]
Yes, the custom styles thing should definitely be re-written and take advantage of webpack `@import` support, that just landed in sass-loader, not a nice solution atm. That should also make it much nicer to override the default vars.

Have to look over it a bit more to see the best way to solve this.

**Comment by eldh** [comment]
Do we need more theme config than what you can set in antwar.config?

**Comment by bebraw** [comment]
> Do we need more theme config than what you can set in antwar.config?

Not sure. I guess we can start from that. No need to be perfect at the first go.

**Comment by eldh** [comment]
I think there's a point in not adding too much complexity.
